BlueprintUI/Sources/Measuring/SizeConstraint.swift

```diff
@@ -11,12 +11,22 @@ public struct SizeConstraint: Hashable {
 
     /// The height constraint.
     public var height: Axis
+    
+    private let hash : Int
```

Comment:
This won't be updated if with or height is mutated

Reply:
Removing this; it's superceded by the caching in the keys anyways